### DISCUSSION TITLE
How to make glass Diagnal

### DISCUSSION THREAD

**Discussion by MaxineMaxie** [BEEmod/BEE2-items#4059] [Tech Support / Q&A] [answered]
I somhow made a pane of glass be at a 45 degree angle and have now clue how I did it, could someone give me more info on this?
![image](https://user-images.githubusercontent.com/94768673/149634573-a654432c-7c2e-4ecb-bd30-cc383c5b30c0.png)

**Comment by TeamSpen210** [comment, marked as answer]
It's an editor bug, it doesn't actually work properly or produce any sensible results.
